nvdaHelper/vbufBackends/gecko_ia2/gecko_ia2.cpp

```diff
@@ -552,14 +558,13 @@ VBufStorage_fieldNode_t* GeckoVBufBackend_t::fillVBuf(IAccessible2* pacc,
 		|| role == ROLE_SYSTEM_BUTTONMENU
 		|| ((role == ROLE_SYSTEM_CHECKBUTTON || role == ROLE_SYSTEM_RADIOBUTTON) && !isLabelVisible(pacc));
 	// Whether this node has a visible label somewhere else in the tree
-	const bool labelVisible = canDetectLabelVisibility // Not all browsers support getting a node's labelledBy node
-		&& nameIsExplicit && name && name[0] //this node must actually have an explicit name, and not be just an empty string
+	const bool labelVisible = nameIsExplicit && name && name[0] //this node must actually have an explicit name, and not be just an empty string
 		&&(!nameIsContent||role==ROLE_SYSTEM_TABLE) // We only need to know if the name won't be used as content or if it is a table (for table summary)
 		&&isLabelVisible(pacc); // actually do the check
 	// If the node is explicitly labeled for accessibility, and we haven't used the label as the node's content, and the label does not visibly appear anywhere else in the tree (E.g. aria-label on an edit field)
 	// then ensure that the label is always reported along withe the node
 	// We must exclude tables from this though as table summaries / captions are handled very specifically
-	if(canDetectLabelVisibility&&nameIsExplicit&&!nameIsContent&&(role!=ROLE_SYSTEM_TABLE)&&!labelVisible) {
+	if(nameIsExplicit&&!nameIsContent&&(role!=ROLE_SYSTEM_TABLE)&&!labelVisible) {

```

Comment:
I understand this is consistent with the style of the rest of the file, however I would find this much easier to read with spaces around the boolean operators.